deployment/ansible/roles/model-my-watershed.app/tasks/dependencies.yml

```diff
@@ -1,12 +1,12 @@
 ---
 - name: Install numba
-  pip: name="{{ item.name }}" version={{ item.version }} state=present
+  pip: name="{{ item.name }}" version={{ item.version }} state=present executable=pip
```

Comment:
Now that we're using the [`ansible_local`](https://www.vagrantup.com/docs/provisioning/ansible_local) provisioner, I'd be curious to see if this is still necessary. It's possible Ansible determined that it needed to use `pip3` because it was configured to use Python 3 on your host.

Reply:
Thanks, removed in 08bb1cc.